app/services/allocation_service.rb

```diff
@@ -153,7 +153,7 @@ def self.current_pom_for(nomis_offender_id, prison_id)
     current_allocation = active_allocations(nomis_offender_id, prison_id)
     nomis_staff_id = current_allocation[nomis_offender_id]['primary_pom_nomis_id']
 
-    PrisonOffenderManagerService.get_pom(prison_id, nomis_staff_id)
+    Prison.new(prison_id).get_pom(nomis_staff_id)
```

Comment:
Could the whole surrounding method move into Prison so we can avoid constructing another one here? Or pass in the prison instead of the prison_id?

Reply:
This is a hint that we have the design wrong - yes we shouldn't be arbitrarily created model objects in this layer